R/render-cached-plot.R

```diff
@@ -471,6 +483,7 @@ renderCachedPlot <- function(expr,
               plotObj = drawReactiveResult,
               width = width,
               height = height,
+              alt = alt,
```

Comment:
`alt` must be retrieved from `altWrapper()` near line 448, similar to `height` and `width` are retrieved from `fitDims`.

Reply:
Thank you! Nice catch! Resolved via c844ea6